### ISSUE TITLE
Can't resolve 'styled-components' in storybook-addon-iframe

### ISSUE THREAD

**Issue by kairos2109** [geometricpanda/geometricpanda#34]
# 🐞 bug report

### Affected Package
@geometricpanda/storybook-addon-iframe

### Description

The dependency `styled-components` does not seem to be included in the .esm.js bundle.

## 🔬 Minimal Reproduction

- Create a project with support for Storybook
- Install and configure the `geometricpanda/storybook-addon-iframe` plugin
- Try to run Storybook locally

## 🔥 Exception or Error
```
ERROR in ./node_modules/@geometricpanda/storybook-addon-iframe/storybook-addon-iframe.esm.js
Module not found: Error: Can't resolve 'styled-components' in '/home/projects/test-storybook/node_modules/@geometricpanda/storybook-addon-iframe'
 @ ./node_modules/@geometricpanda/storybook-addon-iframe/storybook-addon-iframe.esm.js 6:0-41 35:31-39 36:24-32 37:29-37
 @ ./.storybook/preview.js
 @ ./.storybook/preview.js-generated-config-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js
...
```

## 🌍  Your Environment

**Storybook Version:**
<pre><code>
"@storybook/vue": "^6.3.6"
</code></pre>

**@geometricpanda Version:**
<pre><code>
"@geometricpanda/storybook-addon-iframe": "^0.2.1"
</code></pre>

**Comment by ronenaxonius** [comment]
following, just experienced the same issue with storybook (vue)

**Comment by jimdrury** [comment]
Hey, i'll take a look into this this shortly.

As a quick fix you should be able to resolve this by installing styled-components, as a workaround
`npm install styled-components` (or `yarn add styled-components`)

**Comment by jimdrury** [comment]
Should be resolved in 0.2.2
